cves/CVE-2013-0848.yml

```diff
@@ -192,8 +199,9 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: true
+  answer: This could have been found through automatic testing because it could've checked for an
```

Comment:
Another small nit, think you need `|` here or need to put this on one line.

Content-wise I think this is correct, but how would the fuzzer know an out-of-bound array write occurred? (ex: crash?)